src/status_im/contexts/wallet/send/routes/view.cljs

```diff
@@ -241,12 +243,12 @@
                                                            (not-empty routes))]
 
     (rn/use-effect
-     #(when (> (count affordable-networks) 0)
-        (fetch-routes input-value routes-can-be-fetched? 2000))
-     [input-value routes-can-be-fetched?])
+     #(when (and active-screen? (> (count affordable-networks) 0))
+        (fetch-routes input-value valid-input? 2000))
+     [input-value valid-input?])
```

Comment:
Do we actually need to react to `valid-input?`? I think passing `input-value `would be okay

Reply:
Well, theoretically we can remove it because we know that input validity depends only on input value. But that is kind of knowledge that we have outside of the component. If for some reason the validity rule will change, not having valid-input? as a dependency can cause an error